src/openrct2/scripting/bindings/game/ScProfiler.hpp

```diff
@@ -84,13 +84,19 @@ namespace OpenRCT2::Scripting
             OpenRCT2::Profiling::ResetData();
         }
 
+        bool isEnabled() const
+        {
+            return OpenRCT2::Profiling::IsEnabled();
+        }
+
     public:
         static void Register(duk_context* ctx)
         {
             dukglue_register_method(ctx, &ScProfiler::getData, "getData");
             dukglue_register_method(ctx, &ScProfiler::start, "start");
             dukglue_register_method(ctx, &ScProfiler::stop, "stop");
             dukglue_register_method(ctx, &ScProfiler::reset, "reset");
+            dukglue_register_method(ctx, &ScProfiler::isEnabled, "isEnabled");
```

Comment:
Simply "enabled" is typically used for boolean properties in JS.

Reply:
Its a method tho

Reply:
Ah, in that case it'd be more consistent with the rest to make this a property instead.

Reply:
Alright